config/common/translator.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Aliases\Aliases;
+use Yiisoft\Translator\CategorySource;
+use Yiisoft\Translator\Message\Php\MessageSource;
+use Yiisoft\Translator\MessageFormatterInterface;
+
+/** @var array $params */
+
+return [
+    // Configure application CategorySource
+    'translation.app' => static function (Aliases $aliases, MessageFormatterInterface $messageFormatter) use ($params) {
```

Comment:
`MessageFormatterInterface` don't should defined in container. Use concrete implementation specfic to category.